lib/IDE/SyntaxModel.cpp

```diff
@@ -418,6 +418,21 @@ CharSourceRange parameterNameRangeOfCallArg(const TupleExpr *TE,
   return CharSourceRange();
 }
 
+static CharSourceRange findDocCommentRange(DeclAttributes Attrs) {
+  for (auto Attr : Attrs) {
+    if (auto DocAttr = dyn_cast_or_null<RawDocCommentAttr>(Attr)) {
+      return DocAttr->getCommentRange();
```

Comment:
Good catch! @rintaro I cannot think of any reason not using `Decl::getRawComment` here, If we care only attached doc comment in the use case.